CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 0.4.6 - TBD
+
+### Added
+
+- Nothing.
+
+### Changed
+
+- Nothing.
+
+### Deprecated
+
+- Nothing.
+
+### Removed
+
+- Nothing.
+
+### Fixed
+
+- [#46](https://github.com/zendframework/zend-expressive-tooling/pull/46) allows
+  installation of symfony/console ^4.0.
```

Comment:
Hm, I think this entry should be in `Added` section. This PR adds support for symfony/console in v4.